mainwindow.cc

```diff
@@ -3105,7 +3105,17 @@ void MainWindow::hotKeyActivated( int hk )
     toggleMainWindow();
   else
   if ( scanPopup.get() )
+#ifdef HAVE_X11
```

Comment:
The `if` block is no longer one-line, but much longer. Please add `{` above this line and `}` under `#endif` to improve readability and prevent future mistakes.